src/benchmarks/micro/runtime/Bytemark/ByteMark.cs

```diff
@@ -381,6 +382,7 @@ public void BenchNumericSortJagged()
     const int NumericSortRectangularIterations = 5;
 
     [Benchmark]
+    [BenchmarkCategory(Categories.JIT, Categories.ByteMark)]
```

Comment:
>BenchmarkCategory [](start = 5, length = 17)

In other places, I have seen this attribute at class level. Can we do the same for this file as well? Or do they have to be at method level?

Reply:
As long as you can have class level annotations and also method level annotations and they are additive I think that moving it up the class level is fine. @adamsitnik